src/Auth/Basic.php

```diff
@@ -76,19 +76,19 @@ public function register(Hooks $hooks) {
 	}
 
 	/**
-	 * Set cURL parameters before the data is sent
+	 * Set cURL parameters before the data is sent.
 	 *
-	 * @param resource|\CurlHandle $handle cURL handle
```

Comment:
AFAIK, the name of the project is `cURL` => https://curl.se/docs/faq.html#What_is_cURL

Lowercase `curl` would be the shell binary, but the handle here is actually for the integrated `libcurl` library, not for the shell binary.

The class name `CurlHandle` comes from using PascalCase for PHP class names while ignoring brand names and trademarks (like `Mysql` vs `MySQL`).